lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -813,10 +799,7 @@ bool TypeLoweringVisitor::visitStmt(StrictConnectOp op) {
     Value dest = getSubWhatever(op.dest(), field.index());
     if (field.value().isOutput)
       std::swap(src, dest);
-    if (src.getType().isa<AnalogType>())
-      builder->create<AttachOp>(ArrayRef<Value>{dest, src});
-    else
-      builder->create<StrictConnectOp>(dest, src);
+    builder->create<StrictConnectOp>(dest, src);
```

Comment:
StrictConnect handles analog now?  I debated that but wasn't sure which way to go.  Connect implies directionality which analog types don't have.

Reply:
Oh, sorry.  No, they don't handle analog's, neither do non-strict connect's.  The idea is this code is dead because it's iterating over the connect (dest) type, and [strictconnect's verifier reject's if there was an analog in the type](https://github.com/llvm/circt/blob/49bd507ff162ea81aec765869bcf732b1143e217/lib/Dialect/FIRRTL/FIRRTLOps.cpp#L2113).  This was similarly dropped from sibling method `visitStmt(ConnectOp)` [previously](https://github.com/llvm/circt/commit/ba79554f387993c882b649316c17471981046406#diff-0ad6cf1c85d571a808d05866900e638ab97439c724412ce8e14ecfcabba44438L792), FWIW, with similar reasoning.  Sound good, or are these changes problematic?

Reply:
Ah, it's lowering a strict connect.  Sounds good.